build-external-api-jar/project/Dependencies.scala

```diff
@@ -0,0 +1,26 @@
+import sbt._
+
+object Dependencies {
+  val junit = Seq(
+    "junit" % "junit" % "4.12" % Test,
+    ("com.novocode" % "junit-interface" % "0.11" % Test).exclude("junit", "junit-dep")
+  )
+
+  object ideProbe {
+
+    val version = "0.3.0+28-b36b0a51+20210114-1358-SNAPSHOT"
```

Comment:
Is this snapshot available in a public repository?

Reply:
it would be from https://github.com/VirtusLab/ide-probe/commits/master

Reply:
It was built and published here, it's not a local snapshot. I don't know why there's SNAPSHOT suffix in the version name. https://travis-ci.org/github/VirtusLab/ide-probe/jobs/754454201